flake.nix

```diff
@@ -21,7 +21,14 @@
         packages.nixd = nixd;
 
         devShells.default = nixd.overrideAttrs (old: {
-          nativeBuildInputs = old.nativeBuildInputs ++ [ pkgs.clang-tools pkgs.gdb pkgs.nixpkgs-fmt ];
+          nativeBuildInputs = old.nativeBuildInputs ++ [
+            pkgs.clang-tools
+            pkgs.gdb
+            pkgs.nixpkgs-fmt
+          ] ++ [
```

Comment:
how about separate the shell here in something like `devShells.nvim` ?

Reply:
Maybe it's better this way